skops/io/_visualize.py

```diff
@@ -232,6 +232,9 @@ def walk_tree(
             "here: https://github.com/skops-dev/skops/issues"
         )
 
+    if node_name == "constructor":
+        return
```

Comment:
Very sympathetic to the concerns.  Suspect you all know better here, so would defer to your recommendations.

As far as I am aware, sklearn tree-based models are the only ones that produce a "constructor" object of this type (presumably as an artifact of how skops handles C Extension types).

Taking the example:
```python
from sklearn.tree import DecisionTreeClassifier
from skops.io import dumps, loads, visualize
dumped = dumps(DecisionTreeClassifier().fit([[0, 1], [2, 3], [4, 5]], [0, 1, 2]))
loaded = loads(dumped)
visualize(dumped)
```

The skipped node is `<class 'sklearn.tree._tree.Tree'>`. Skipping processing of this object, the code outputs:

> root: sklearn.tree._classes.DecisionTreeClassifier
>└──
> attrs: builtins.dict
>    
>├──
> criterion: json-type("gini")
>    
>├──
> splitter: json-type("best")
>    
>├──
> max_depth: json-type(null)
>    
>├──
> min_samples_split: json-type(2)
>    
>├──
> min_samples_leaf: json-type(1)
>    
>├──
> min_weight_fraction_leaf: json-type(0.0)
>    
>├──
> max_features: json-type(null)
>    
>├──
> max_leaf_nodes: json-type(null)
>    
>├──
> random_state: json-type(null)
>    
>├──
> min_impurity_decrease: json-type(0.0)
>    
>├──
> class_weight: json-type(null)
>    
>├──
> ccp_alpha: json-type(0.0)
>    
>├──
> n_features_in_: json-type(2)
>    
>├──
> n_outputs_: json-type(1)
>    
>├──
> classes_: numpy.ndarray
>    
>├──
> n_classes_: numpy.int64
>    
>├──
> max_features_: json-type(2)
>    
>├──
> tree_: sklearn.tree._tree.Tree
>    │   
>├──
> attrs: builtins.dict
>    │   │   
>├──
> max_depth: json-type(2)
>    │   │   
>├──
> node_count: json-type(5)
>    │   │   
>├──
> nodes: numpy.ndarray
>    │   │   
>└──
> values: numpy.ndarray
>    │   
>├──
> args: builtins.tuple
>    │   │   
>├──
> content: json-type(2)
>    │   │   
>├──
> content: numpy.ndarray
>    │   │   
>└──
> content: json-type(1)
>    
>└──
> _sklearn_version: json-type("1.3.0")

Reply:
I see. I think the best solution here would be to actually handle `type`, as in, in the visualized tree print what the type is, rather than skipping it. I'm sure there would be other cases where we'd encounter types as well.

Reply:
Gave this a first pass. So if `isinstance(node, type)`, then print information about the type rather than skipping.